rero_ils/modules/items/api/issue.py

```diff
@@ -47,3 +52,195 @@ def issue_is_regular(self):
     def issue_status_date(self):
         """Shortcut for issue status date."""
         return self.get('issue', {}).get('status_date')
+
+    @property
+    def claims_count(self):
+        """Shortcut for the issue claims_count."""
+        return self.get('issue', {}).get('claims_count', 0)
+
+    @property
+    def vendor_pid(self):
+        """Shortcut for vendor pid if exists."""
+        from ...holdings.api import Holding
+        if self.holding_pid:
```

Comment:
`holding_pid` comes from `ItemRecord` not easy to understand at this place ?
Actually this property is not working in this class.

Little thing I found out, `return None` could be deleted because a function without return returns `None`.